content/blog/2017/2017-03-12-toulousejam-pipeline-workshop.adoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Pipeline Workshop & Hackergarten @ ToulouseJAM Feedback"
+tags:
+- hackergarten
+- jam
+- meetup
+- feedback
+author: batmat
+imagesdir: /images/post-images/2017-02-toulousejam-workshop
+---
+
+Last week, a full-day event about Jenkins Pipeline was organized in link:https://www.google.fr/maps/place/Toulouse/@43.6006786,1.3628012,12z/data=!3m1!4b1!4m5!3m4!1s0x12aebb6fec7552ff:0x406f69c2f411030!8m2!3d43.604652!4d1.444209[Toulouse, France] with the link:https://www.meetup.com/fr-FR/Toulouse-Jenkins-Area-Meetup/events/237089783/[Toulouse JAM].
```

Comment:
"Last week"

Note: it's taken me quite some more time to finish this that I planned, so the date and this will be probably plain wrong when reviewed, I considered fixing it just now, but better do it when review is almost done I guess. I expect quite some back & forth :-).